template/php7/readme.md

```diff
@@ -0,0 +1,37 @@
+# PHP7 Template
+
+Templates are built using the latest minor version of the major release.
+
+| Modules (by default) |
+| ------------- |
+| Core, date, libxml, openssl, pcre, sqlite3, zlib, ctype, curl, dom, fileinfo, filter, ftp, hash, iconv, json, mbstring, SPL, PDO, pdo_sqlite, session, posix, readline, Reflection, standard, SimpleXML, Phar, tokenizer, xml, xmlreader, xmlwriter, mysqlnd, sodium |
+
+## Usage:
+
+```shell
+faas-cli new my-function-php7 --lang php7
+```
+
+You will find in the newly created directories `my-function-in-php5` and `my-function-in-php7`:
+- `src/Handler.php` : entrypoint
+- `src/php-extension.sh` : is for installing PHP extensions if needed
+- `src/composer.json` : is for dependency management
+
+## Extra Extensions
+
+If you need to install Phalcon for example, check out the following gist - you can use
+that file in your function into your `src/php-extension.sh` file;
+
+https://gist.github.com/andrew-s/07e152b7f981e8453da8d2d991d9aab4
```

Comment:
I'd prefer this be checked in as `php-extension.sh-example` and linked here.

Reply:
Sure, I'm unsure how much this would be used but, I'd imagine the default extensions probably suffice for most people anyway. Do we agree on renaming?

Reply:
Are you suggesting to putting this into `php-extensions.sh` and renaming that file into `.sh-example`? or just the first?

Reply:
Nope, I suggest putting in a file `template/php7/php-extensions.sh-example`, ie one in the repo, which can be linked to from the README.md in the same dir, and won't appear in the templated function that gets created on `new`, make sense?

Rationale is to keep everything in one place, much easier to update and manage.